docs/review-template.md

```diff
@@ -205,10 +206,10 @@ Regular presentations at conferences by the community are OPTIONAL. |  |
 
 Requirement | meets | &nbsp;links&nbsp;and&nbsp;notes&nbsp;
 -----|-----|-----
-Contributions MUST adhere to either a coding or writing style guide, either the [codebase](../glossary.md#codebase) community's own or an existing one that is advertised in or part of the codebase. |  |
+The [codebase](../glossary.md#codebase) MUST use a coding or writing style guide, either the codebase community's own or an existing one referred to in the codebase. |  |
 Contributions SHOULD pass automated tests on style. |  |
-The codebase SHOULD include inline comments and documentation for non-trivial sections. |  |
-Including sections on [understandable English](https://standard.publiccode.net/criteria/understandable-english-first.html) in the style guide is OPTIONAL. |  |
+The style guide SHOULD include expectations on inline comments and documentation for non-trivial sections. |  |
```

Comment:
```suggestion
The style guide SHOULD include expectations for inline comments and documentation for non-trivial sections. |  |
```

Reply:
The review template is generated; I'll double check that it generates this correctly based on the upstream change of the actual requirement text